src/app.rs

```diff
@@ -211,6 +211,8 @@ impl Application {
                 tokio::select! {
                 Some(signal) = signals.next() => {
                     if signal == SignalTo::Reload {
+                        // Reload paths
+                        config_paths = config::process_paths(&opts.config_paths).unwrap_or(config_paths);
```

Comment:
All of these changes are in the master, it was probably lost in a bad merge.

Reply:
@oktal and I were just ironing those out. :)